crates/test_utils/wasm/wasm_workspace/create_entry/src/lib.rs

```diff
@@ -56,6 +56,11 @@ fn create_post(post: crate::Post) -> ExternResult<HeaderHash> {
     hdk::prelude::create_entry(&post)
 }
 
+#[hdk_extern]
+fn delete_post(post_hash: HeaderHash) -> ExternResult<HeaderHash> {
```

Comment:
i can't see this being referenced anywhere in this PR which is confusing. can you explain where it's used or why it's needed?

Reply:
It's to make sure the wasm can compile with the given input types for `delete_entry`